src/collective/lazysizes/configure.zcml

```diff
@@ -21,11 +21,18 @@
       permission="collective.lazysizes.Setup"
       />
 
+  <!-- zope resource files -->
   <browser:resourceDirectory
       name="collective.lazysizes"
       directory="static"
-      layer="collective.lazysizes.interfaces.ILazySizesLayer"
```

Comment:
is this because a bug in Plone 5?

Reply:
first i tried to use the ++resource++... resources, which does not work with bundles (they need to be able to be stored in the ZODB) - therefore I switched to plone.resource directories. so this change might not be needed, but doesn't hurt also. e.g. the controlpanel references a image via the resource directory path...

Reply:
this is a know bug in Plone 5: https://github.com/plone/Products.CMFPlone/issues/1749

Reply:
Commented on that: https://github.com/plone/Products.CMFPlone/issues/1749#issuecomment-332007871